presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3StorageClass.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.s3;
+
+public enum PrestoS3StorageClass {
+    STANDARD, IntelligentTiering,
```

Comment:
```suggestion
    STANDARD, 
    INTELLIGENTTIERING,
    /**/
```

Reply:
It actually should be `INTELLIGENT_TIERING`. 

You can then implement `fromString` method to allow users to write `IntelligentTiering` in the config file.
Example: https://github.com/airlift/airlift/blob/675dfe92e23d4b341a8b3b00c28d480ec517ef80/configuration/src/test/java/io/airlift/configuration/TestConfigurationFactory.java#L682-L699

Reply:
Let's call it `INTELLIGENT_TIERING` without any special `fromString` handling, as that's what Amazon calls it in the documentation: https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html

> STANDARD—The default storage class. If you don't specify the storage class when you upload an object, Amazon S3 assigns the STANDARD storage class.

> The INTELLIGENT_TIERING storage class is designed to optimize storage costs by automatically moving data to the most cost-effective storage access tier, without performance impact or operational overhead.